src/Adapter/Translations/TranslationRouteFinder.php

```diff
@@ -203,7 +204,10 @@ public function findRouteParameters(ParameterBag $query)
     private function isModuleUsingNewTranslationSystem($moduleName)
     {
         $module = $this->moduleRepository->getInstanceByName($moduleName);
+        if (Validate::isLoadedObject($module)) {
```

Comment:
I think this check would be more adequate:

```suggestion
        if ($module instanceof Module) {
```